configurate-toml/src/main/java/ninja/leaping/configurate/toml/TOMLOptions.java

```diff
@@ -0,0 +1,140 @@
+/*
+ * Configurate
+ * Copyright (C) zml and Configurate contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package ninja.leaping.configurate.toml;
+
+import com.moandjiezana.toml.TomlWriter;
+import org.checkerframework.checker.nullness.qual.NonNull;
+
+import java.time.ZoneOffset;
+import java.util.TimeZone;
+
+/**
+ * Represents various options for TOML saving.
+ */
+public class TOMLOptions {
```

Comment:
IMO, the options encapsulated by this class should just be expressed on the loader. This is the layout used by all of the other loader impls.

I've done it this way in my implementation here.
https://github.com/lucko/configurate/blob/lucko-master/configurate-toml/src/main/java/ninja/leaping/configurate/toml/TOMLConfigurationLoader.java#L62-L172

Feel free to copy.